circle.yml

```diff
@@ -236,8 +236,8 @@ commands:
         type: string
     steps:
       - run:
-          name: "Upgrade codecov"
-          command: sudo pip3 install --upgrade --quiet --no-cache-dir codecov
+          name: "Install codecov"
+          command: pip3 install --break-system-packages --upgrade --user --quiet --no-cache-dir codecov
```

Comment:
What is this setting doing?

Reply:
Installs codecov python script. In the new version of pip/Debian it is disallowed to install pip packges "globally". I have not found better way than forcing this with `--break-system-packages`.

Reply:
> I have not found better way than forcing this with --break-system-packages.

😬